preprocessors/celebrity-detector/celebrity-detector.py

```diff
@@ -214,7 +214,8 @@ def categorise():
         except jsonschema.exceptions.ValidationError as e:
             logging.error(e)
             return jsonify("Invalid Preprocessor JSON format"), 500
-        logging.debug("Sending response")
+        logging.debug("Celebrity Detector Response")
+        logging.debug(response)
```

Comment:
again potential privacy issue?

Reply:
Possibly...maybe something like "detected [n] celebrities out of [m] objects"?